### DISCUSSION TITLE
Crystal

### DISCUSSION THREAD

**Discussion by Graemec** [hanamimastery/episodes#9] [Suggestions]
Do you have any plans to make Hanami usable by Crystal?

**Comment by swilgosz** [comment]
Not sure what you mean by that. Hanami can be integrated with any frontend framework by configuring assets properly.

I plan to show a few episodes about views and assets, integrating different solutions, but that's rather further in the future, because I'm 90% ruby developer and it would be extremely hard for me to do research efficiently.

However, on Hanami Mastery we welcome also external content contributions. If anyone would like to help by writing an article about such integration, we can use our studio to transform that into an episode :).
